tests/www/views/test_views_rendered.py

```diff
@@ -250,6 +254,13 @@ def test_rendered_template_secret(admin_client, create_dag_run, task_secret):
     assert ti.state == TaskInstanceState.QUEUED
 
 
+if os.environ.get("_AIRFLOW_SKIP_DB_TESTS") == "true":
```

Comment:
This test needs database initialized when collection happens. Not perfect, but it shows an easy way how you can handle it in cases where you want to get --skip-db-tests still working and parsing this file while collecting.